data/usp_drug_classification_tidying_script.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Parse and tidy the USP Drug Classification data from KEGG
+Raw data from http://www.genome.jp/kegg-bin/get_htext?htext=br08302.keg
+
+Created on Sun Feb  5 13:15:04 2017
+
+@author: claire
+"""
+
+import numpy as np
+import pandas as pd
+
+if __name__ == "__main__":
+    
+    fname = 'br08302.keg'
```

Comment:
Is this data on `data.world`? It might be worth not assuming that the data exists on the local system, or at least check whether it exists on the local system. 
There's a function in `scripts/read_data.py` that might make that easier (you might need to `git pull upstream/master`).